NewDawn/NewDawn/ViewControllers/ChatPageViewControllers/ChatRoomViewController.swift

```diff
@@ -202,6 +202,29 @@ class ChatRoomViewController: MessagesViewController {
         messagesCollectionView.messageCellDelegate = self
         messageInputBar.delegate = self
         fetchMessagesFromHistory()
+        fetchEndUserProfile() {
+            profile in
+            if profile.takenRequestedFromYou {
```

Comment:
Have you print to see if this if block went through?

Reply:
How to print it? I only tried to put it to the else part and in every chatpage it will display the alert below, even though nobody sent taken request to me.
<img width="340" alt="Screen Shot 2019-06-12 at 9 29 32 PM" src="https://user-images.githubusercontent.com/49891341/59402911-0a2f5200-8dd3-11e9-9741-459c64e5bd4f.png">
So I think I may put it in the if part, and currently the alert will not show (but I guess it's correct, since no one sent taken request to me, so I should not get this alert)